packages/writing-style/bin/commercetools-vale.js

```diff
@@ -1,25 +1,11 @@
 #!/usr/bin/env node
 
-const os = require('os');
 const path = require('path');
-const shelljs = require('shelljs');
-const { valeVersion } = require('../package.json');
+const run = require('../src/run');
 
-const binaryFileName =
-  os.platform() === 'win32' ? `vale-${valeVersion}.exe` : `vale-${valeVersion}`;
-
-const valeBinary = path.join(__dirname, `./${binaryFileName}`);
 const valeConfig = path.join(__dirname, `../.vale.ini`);
-let params = process.argv.slice(2);
-// simplistic check whether the --config param was passed empty by the VSCode plugin (empty = followed by another flag)
-const configParamPosition = params.findIndex((p) => p === '--config');
-if (
-  params[configParamPosition + 1] &&
-  params[configParamPosition + 1].startsWith('-')
-) {
-  params = params.filter((p) => p !== '--config');
-}
```

Comment:
As far as I understand, this is not necessary anymore, as for VSCode we can use the "config" option of the plugin and use the `vale-bin` binary.